tox.ini

```diff
@@ -11,7 +11,8 @@ skipsdist = True
 install_command =  pip install {opts} {packages}
 commands = py.test {posargs}
 deps =
-    {toxinidir}
+    -e{toxinidir}
+    --no-index
```

Comment:
NB: This is needed to work around juju-solutions/python-libjuju#2.  If it's not included, then re/building the tox env tries to pull in the conflicting juju package due to the higher version number than what's in the wheelhouse.